stargazer/stargazer.py

```diff
@@ -314,19 +326,32 @@ def generate_footer_html(self):
 
         if not self.show_footer:
             return footer
+        if 'footer top' in self.custom_lines:
+            footer += self.generate_custom_lines_html('footer top')
         footer += self.generate_observations_html()
         footer += self.generate_r2_html()
         footer += self.generate_r2_adj_html()
         if self.show_residual_std_err:
             footer += self.generate_resid_std_err_html()
         if self.show_f_statistic:
             footer += self.generate_f_statistic_html()
+        if 'footer bottom' in self.custom_lines:
+            footer += self.generate_custom_lines_html('footer bottom')
         footer += '<tr><td colspan="' + str(self.num_models + 1) + '" style="border-bottom: 1px solid black"></td></tr>'
         footer += self.generate_notes_html()
         footer += '</table>'
 
         return footer
 
+    def generate_custom_lines_html(self, location):
+        custom_text = ''
+        for custom_row in self.custom_lines[location]:
+            custom_text += '<tr><td style="text-align: left">' + str(custom_row[0]) + '</td>'
```

Comment:
This ``str`` should in principle be replaced by the appropriate formatters for ``float`` (``_float_format``) and some day, ``int`` (see #53); ideally, ``add_line`` should even have an optional ``formatter`` argument. But all of this can be done later in a separate PR to now slow down this one.